log4j-changelog/src/main/resources/log4j-changelog.xsd

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements. See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License. You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<schema xmlns="http://www.w3.org/2001/XMLSchema"
+  targetNamespace="https://logging.apache.org/log4j/log4j-changelog"
+  xmlns:cl="https://logging.apache.org/log4j/log4j-changelog" elementFormDefault="qualified">
+
+  <element name="release">
+    <complexType>
+      <simpleContent>
+        <extension base="string">
+          <attribute name="version" use="required"/>
+          <attribute name="date" use="required">
+            <simpleType>
+              <union memberTypes="date cl:tbd"/>
+            </simpleType>
+          </attribute>
+        </extension>
+      </simpleContent>
+    </complexType>
+  </element>
+
+  <simpleType name="tbd">
+    <restriction base="string">
+      <enumeration value="TBD"/>
+    </restriction>
+  </simpleType>
+
+  <element name="entry">
+    <complexType>
+      <sequence>
+        <element name="issue" type="cl:issueType" minOccurs="0" maxOccurs="unbounded"/>
+        <element name="author" type="cl:authorType" maxOccurs="unbounded"/>
+        <element name="description" type="cl:descriptionType"/>
+      </sequence>
+      <attribute name="type" type="cl:changeType" use="required"/>
+    </complexType>
+  </element>
+
+  <complexType name="issueType">
+    <simpleContent>
+      <extension base="string">
+        <attribute name="id" type="ID" use="required"/>
```

Comment:
Judging from our conversation with you, `type` needs to be replaced with `string` here, since IDs in GitHub Issues are positive integers.

```suggestion
        <attribute name="id" type="string" use="required"/>
```